src/program/services/streaming/media_stream.py

```
@@ -0,0 +1,1194 @@
+                                                #
+                                                # Returning the wrong data would be catastrophic for playback,
+                                                # so we force a reconnect at the correct position.
+                                                connection.seek(position=chunk.start)
+
+                                            chunk_label = f"[{chunk.start}-{chunk.end}]"
+
+                                            with benchmark(
+                                                log=lambda duration: logger.log(
+                                                    "STREAM",
+                                                    self._build_log_message(
+                                                        f"Fetching {chunk} took {duration}s"
```

Ruff (0.14.3)

Function definition does not bind loop variable `chunk`

(B023)